packages/web/src/services/solana/solana.ts

```diff
@@ -4,17 +4,12 @@ import {
   getMinimumBalanceForRentExemptAccount,
   getAssociatedTokenAddressSync
 } from '@solana/spl-token'
-import {
-  PublicKey,
-  Transaction,
-  TransactionInstruction,
-  Keypair
-} from '@solana/web3.js'
+import { PublicKey, Transaction, Keypair } from '@solana/web3.js'
 
 import { getLibs } from 'services/audius-libs'
 
 export const ROOT_ACCOUNT_SIZE = 0 // Root account takes 0 bytes, but still pays rent!
-export const TRANSACTION_FEE_FALLBACK = 5000
+export const TRANSACTION_FEE_FALLBACK = 10000
```

Comment:
Was seeing some transactions have a fee of this, wanted to take the max of what we saw vs the min to ensure success (if this was too low, and the transaction was actually higher in fees, the withdrawal would fail). I think it's better to err on the side of using too much USDC to do the swap/create than having the withdrawal fail.